doc/tutorial/iris-dnn.md

```diff
@@ -26,13 +26,13 @@ limit 5;
 
 ## Train
 
-Let's train a DNNClassifier, which has two hidden layers where each layer has ten hidden units. This can be done by specifying the training clause for SQLFlow's extended syntax.
+Let's train a 3-class DNNClassifier, which has 2 hidden layers with 10 hidden units each. This can be done by specifying the training clause for SQLFlow's extended syntax.
```

Comment:
Use English than numbers in formal documents should be better.

Reply:
Done.